src/index.html

```diff
@@ -2,20 +2,67 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
-      name="viewport"
-      content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+    <meta name="viewport" content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
-      "
-    >
+      ">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img class="card__image"
+        src="./images/imac.jpeg"
+        alt="img"
+      >
+
+      <p class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__model">
+        Product code: 195434
+      </p>
+
+      <div class="review">
+        <div class="review__flex">
+          <div class="review__stars review__stars--1"></div>
+
+          <div class="review__stars review__stars--2"></div>
+
+          <div class="review__stars review__stars--3"></div>
+
+          <div class="review__stars review__stars--4"></div>
+
+          <div class="review__stars review__stars--5"></div>
+        </div>
+
+        <p class="review__text">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card-price">
+        <p class="card-price__text">
+          Price:
+        </p>
```

Comment:
`p` is not a correct tag here because semantically `p` means a complete piece of information but `Price:` is just a part of a message

So use `p` for entire block but here you need a `span`